style.css

```diff
@@ -484,7 +484,258 @@ footer {
+------------------------------------------------- */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  /* header/ navbar media query start */
+
+  .navbar-container {
+    min-height: 72px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .nav-ul {
+    width: 50%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .nav-ul li a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: var(--color-white);
+  }
+
+  /* header/ navbar media query end */
+
+  /* headline/homepage media query star */
+
+  .section {
+    min-height: 500px;
+    background-color: var(--color-black);
+    background-image: url("./images/desktop-1.svg");
+    background-repeat: no-repeat;
+    background-size: 50%;
+    background-position-x: 100%;
+    background-position-y: 90%;
+  }
+
+  .headline-container {
+    width: 50%;
+    flex-direction: row-reverse;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 85%;
+    height: 100%;
+    padding-right: 10%;
+  }
+
+  .headline-text h1 {
+    font-size: 50px;
+  }
+
+  .headline-text h3 {
+    font-size: 25px;
+  }
+
+  .social-media-icons {
+    width: 15%;
+    margin-top: 25%;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  /* work section media query for desktop version */
+
+  .works-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-title-dv {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .works-title {
+    font-size: 15px;
+    line-height: 20px;
+    margin: 2px;
+    margin-top: 55px;
+    width: 15%;
+  }
+
+  .work-title-dv hr {
+    margin: 0;
+    width: 85%;
+    height: 1px;
+    background-color: #0d151e;
+  }
+
+  .feature-area {
+    min-width: 750px;
+    display: flex;
+    align-items: center;
+  }
+
+  .feature-image {
+    width: 60%;
+    height: 448px;
+  }
+
+  .post-text {
+    width: 40%;
+    height: 300px;
+    padding: 10px;
+    padding-bottom: 100px;
+  }
+
+  .title-post {
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .post-description {
+    font-size: 15px;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .tag-ul-1 {
+    display: flex;
+    justify-content: space-around;
+    width: 70%;
+    height: 10%;
+    margin-bottom: 6px;
+  }
+
+  .tag-ul-1 li {
+    font-size: 10px;
+    line-height: 4px;
+    padding: 12px 8px;
+  }
+
+  .post-text button {
+    padding: 6px 6px;
+    width: 10%;
+    font-size: 15px;
+    margin-top: 12px;
+  }
+
+  .work-cards {
+    height: 100%;
+    width: 100%;
+    display: grid;
+    grid-template: repeat(2, minmax(450px, 1fr)) / repeat(3, minmax(200px, 1fr));
+    grid-gap: 24px;
+  }
+
+  .card {
```

Comment:
- [ ] I think it would be better to add more sides margins to these cards in order to match the Figma design :recycle: 

- Your design:
![Screenshot from 2023-01-26 02-00-18](https://user-images.githubusercontent.com/97834160/214721519-00971ac8-e804-4f43-b1a4-5a124495860c.png)

- Figma design:
![Screenshot from 2023-01-26 02-00-39](https://user-images.githubusercontent.com/97834160/214721715-f059faef-a371-43b5-a24a-7f0856bac892.png)